packages/sdk-codegen/src/codeGen.ts

```diff
@@ -652,6 +659,7 @@ export abstract class CodeGen implements ICodeGen {
   arrayClose = ']'
   hashOpen = '{'
   hashClose = '}'
+  hashKeyQuote = ''
```

Comment:
is this mean to be an empty string? or did you mean `'"'` or conversely `"'"`?

Reply:
oh, nm - javascript objects aren't quoted

Reply:
Should be empty string in the base class